rust/client-tunnel/Cargo.toml

```diff
@@ -6,6 +6,14 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+anyhow = "1.0.81"
+nix = { version =  "0.28.0", features = ["user"] }
+serde = { version = "1.0.197", features = ["derive"] }
+serde_json = "1.0.114"
+tokio = { version = "1.36.0", features = ["fs", "io-util", "macros", "net", "rt-multi-thread", "time"] }
```

Comment:
It might be good to be more granular with these dependencies if they aren't required for other platforms, as our sleuthing session today discovered.

Reply:
Yeah. If `version-check` has gone back to Linux or been removed completely, this doesn't have to compile on macOS at all, just Windows and Linux. So I can move `nix` into a Linux section for sure. I'll look into it.